prog/vm/nexus.rb

```diff
@@ -112,7 +120,9 @@ def allocate
 
   def start
     vm_host_id = allocate
-    vm.update(vm_host_id: vm_host_id, ephemeral_net6: VmHost[vm_host_id].ip6_random_vm_network.to_s)
+    vm.update(vm_host_id: vm_host_id, ephemeral_net6: VmHost[vm_host_id].ip6_random_vm_network.to_s,
+      ephemeral_net4: VmHost[vm_host_id].ip4_random_vm_network.to_s,
+      local_vetho_ip: VmHost[vm_host_id].vetho_ip4_local_random_vm_network.to_s)
```

Comment:
I think it sends 3 different SQL query.
```suggestion
    vmh = VmHost[vm_host_id]
    vm.update(vm_host_id: vm_host_id, ephemeral_net6: vmh.ip6_random_vm_network.to_s,
      ephemeral_net4: vmh.ip4_random_vm_network.to_s,
      local_vetho_ip: vmh.vetho_ip4_local_random_vm_network.to_s)
```